src/main/java/org/folio/circulation/domain/RequestQueueRepository.java

```diff
@@ -48,6 +48,19 @@ public static RequestQueueRepository using(Clients clients) {
       .thenApply(r -> r.map(RequestQueue::new));
   }
 
+  public CompletableFuture<Result<RequestQueue>> getLiteRequestQueues(String itemId) {
```

Comment:
does `lite` have a special meaning? It isn't a term I've encountered in a context like this before.

Reply:
I was a bit confused by this too. I thought it was a misspelling, but, after digging, I realized it was about less stuff being done. I blame Miller for the popularity of "lite" thanks to its Lite beer ads from the past 4 decades. Maybe rename this to `getRequestQueueWithoutItemLookup`. That's a mouthful. Initially, I was more confused about `findByLite` below. I thought "lite" was some sort of index to look up, something like `findByBarcode`.

Reply:
I like this, getRequestQueueWithoutItemLookup, @mreno-EBSCO